tests/unit/customizations/test_paginate.py

```diff
@@ -350,3 +353,23 @@ def test_can_handle_missing_page_size(self):
         del self.parsed_args.page_size
         self.assertIsNone(paginate.ensure_paging_params_not_set(
             self.parsed_args, {}))
+
+class TestNonPositiveMaxItems:
+    @pytest.fixture
+    def max_items_page_arg(self):
+        return PageArgument('max-items', 'documentation', int, 'MaxItems')
```

Comment:
I meant we can put this at the top of the module. The nice thing about fixtures is that `pytest` has a predictable ordered list of locations where it looks for fixtures, so they don't need to be tied down to test classes. Keeping this at the top will make it more reusable. I know there's a question of whether or not this fixture will be used by other tests in the future, but I can imagine generalizing this to be a `page_arg` fixture that acts as sort of a factory for returning different `PageArgument` instances.